src/App.tsx

```diff
@@ -10,33 +13,25 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState<string>('tab-1');
```

Comment:
What is 'tab-1'?
What would be selected if I have another array of tabs with other values in the property id?